roihunter.php

```diff
@@ -52,7 +52,7 @@ class Roihunter extends Module {
     private $rhRequestsManager;
 
     public function __construct() {
-        $this->name = self::ROI_HUNTER_MODULE_NAME;
```

Comment:
@Vergil333 Is the validator show an error if we use a constant here?

Reply:
@roma-kisel yes :) Validator is probably expecting string and can't work with constant.